packs/spell-effects/spell-effect-courageous-anthem.json

```
@@ -53,6 +53,11 @@
                     }
                 ],
                 "selector": "strike-damage"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "option": "composition-signature:courageous-anthem"
```

```suggestion
                "option": "courageous-anthem:origin:signature:{item|origin.signature}"
```
Have to actually include the signature.